blog/2023/07/high-availability-for-klaw.md

```diff
@@ -4,7 +4,7 @@ title: High availability for Klaw
 authors: muralibasani
 tags:
   [
-    kafka,
+    Apache Kafka,
```

Comment:
This one does not need to be replaced, since it does represent a tag, so the capitalisation is not important. You could do this:

```suggestion
    kafka,
    apache-kafka,
```

Reply:
ook